app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -316,6 +320,17 @@ class ChooseStep extends React.Component {
       disabledMessage = `To download ${
         downloadType.display_name
       }, you must be the original uploader of all selected samples.`;
+    } else if (
+      downloadType.type === "original_input_file" &&
+      appConfig.maxSamplesBulkDownloadOriginalFiles &&
```

Comment:
don't think you need this line

Reply:
My thinking was that there's a chance (however small) that `appConfig. maxSamplesBulkDownloadOriginalFiles` is undefined, and I'd rather make it explicit what the behavior is in that case rather than relying on the code to handle `undefined` correctly. Does that sound reasonable?